extensions/standard-processors/processors/ExtractText.h

```diff
@@ -36,26 +36,43 @@ namespace minifi {
 namespace processors {
 
 //! ExtractText Class
-class ExtractText : public core::Processor {
+class
+ExtractText : public core::Processor {
```

Comment:
I think this strange formatting was done by mistake.

Reply:
yes it was, thanks for catching it -- fixed in f4f8c9a11f2fd379c3b5593441619ad547736246